Tests/AlicerceTests/Network/MockNetworkStack.swift

```diff
@@ -29,8 +29,7 @@ final class MockNetworkStack: NetworkStack {
     var beforeFetchCompletionClosure: (() -> Void)?
     var afterFetchCompletionClosure: (() -> Void)?
 
-    init(mockData: Data?, mockError: Network.Error?, queue: DispatchQueue = DispatchQueue.global()) {
-        precondition(mockData != nil || mockError != nil)
+    init(mockData: Data? = nil, mockError: Network.Error? = nil, queue: DispatchQueue = DispatchQueue.global()) {
```

Comment:
if both `mockData` and `mockError` are `nil`, this doesn't make much sense right? (that's why there was a `precondition` on the `init`)

(it will also crash when used 😆, see below)

Since they are both `var`'s, perhaps it's better to simply remove them from the `init`, since they are both initialized as `nil` anyway.

As mentioned above, it will crash when not configured correctly, so it's fine I guess.

Reply:
Agree. Will remove the `var`'s from `init`.